internal/expanders/expanders.go

```diff
@@ -0,0 +1,19 @@
+// Copyright Contributors to the Open Cluster Management project
+package expanders
+
+func GetExpanders() map[string]Expander {
+	return map[string]Expander{
+		"kyverno": KyvernPolicyExpander{},
+	}
+}
+
+type Expander interface {
+	CanHandle(manifest map[string]interface{}) bool
+	Expand(manifest map[string]interface{}, severity string) []map[string]map[string]interface{}
+}
```

Comment:
Ah, interface. Nice. Yes, this is what it should be. 🙂 

I was wondering: Do we want to add something like an `ExpandSetting` string to the interface and then use loops to iterate over each setting for the policyDefault object and policies array? In this case, it'd be "InformKyvernoPolicies" and then the logic for expansion can truly only be in the `expanders.go` file. Or is that going overboard?

Reply:
Okay, so the `Expander` would have to be a Struct to do that...

Reply:
It's a good idea. I'll add a method that checks if the expander is enabled for the policy.